pkg/cli/admin/mustgather/summary.go

```diff
@@ -0,0 +1,179 @@
+package mustgather
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/docker/go-units"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/config/clusteroperator/v1helpers"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// PrintBasicClusterState prints a human-readable highlight of some basic information about the openshift cluster that
+// is valuable to every caller of `oc must-gather`.  Even different products.
+// This is NOT a place to add your pet conditions.  We have three places for components to report their errors,
+//  1. clusterversion - shows errors applying payloads to transition versions
+//  2. clusteroperators - shows whether every operand is functioning properly
```

Comment:
Should the summary - show what operators are installed (IE: in addition to the core payload (from olm)? 

I ask this question (not to inflate the summary) but have we / are we considering how to display 'team specific summaries' of information that would not be surfaced by these things. 

Its often useful to know whats in/running on a cluster to look for known things that conflict/collide with one another. IE: the way we have this scope (today) may be too narrow and we may want to make room for people to provide high level summary/surfacing information to this.